packages/api/lib/writeRecords/write-granules.js

```diff
@@ -180,13 +180,15 @@ const _writePostgresGranuleViaTransaction = async ({
   executionCumulusId,
   trx,
   granulePgModel,
+  writeConstraints = true,
```

Comment:
Can we add this new parameter to the JSDoc?

Reply:
1c01b68253

Reply:
1c01b68253